src/material/list/list.ts

```diff
@@ -204,6 +213,14 @@ export class MatListItem extends _MatListItemMixinBase implements AfterContentIn
     }
   }
 
+  /** Whether the option is disabled. */
+  @Input()
+  get disabled() { return this._disabled || !!(this._list && this._list.disabled); }
```

Comment:
Should we limit this to list-items inside a nav-list or action-list? For standard text list-items the term `option` (as in the JSDoc) seems incorrect and disabling seems weird too.

This is what Jeremy wrote in the past: https://github.com/angular/components/pull/3624#issuecomment-301623063 and this would match my expectation that we only allow disabled for such list items. Not sure how can we achieve this, but at least by mentioning it in the jsdoc, or by throwing if this changes for standard list items?

Reply:
It's mostly a visual thing anyway so there shouldn't be any harm if somebody ends up doing it for a plain list. Having it be limited to nav/action lists would mean to create dedicated components for those lists or throwing in the setter here which seems unnecessary.

Reply:
Yeah I never proposed that we should have separate components for items of either nav-list, action-list or standard list. Though I don't see anything wrong with throwing. I don't feel strong about it either. At least though I'd expect that we make it clear in the JSDoc. `"option"` seems to be ambiguous and not clear in the context of a list-item.